server/tests/backends/fixtures/convert/to_text_pypika.json

```diff
@@ -0,0 +1,171 @@
+{
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake",
+    "athena_pypika"
```

Comment:
what's the matter with athena ?

Reply:
Rounding difference: 

```E   DataFrame.iloc[:, 1] (column name="price_per_l") values are different (10.0 %)
 [index]: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
 [left]:  [0.0559999999999999, 0.045, 0.065, 0.005, 0.06, 0.07, 0.054, 0.084, 0.16, 0.005]
 [right]: <StringArray>
 ['0.056', '0.045', '0.065', '0.005',  '0.06',  '0.07', '0.054', '0.084',  '0.16', '0.005']
  Length: 10, dtype: string
```

Reply:
I don't know :/ But I think there's also a type problem: Here, `left` seems to contain floats whereas right contains strings... I'll have a look locally